openassessment/xblock/static/js/src/studio/oa_container_item.js

```diff
@@ -112,6 +114,20 @@ OpenAssessment.Prompt.prototype = {
         return OpenAssessment.Fields.stringField(sel, text);
     },
 
+    /**
+     *  Get prompt type from current textarea.
+     *
+     * @returns {string}
+     */
+    promptType: function() {
+        var promptType = $('.openassessment_prompt_description', this.element).data('w-type');
```

Comment:
please use return instead of making an extra variable.

Reply:
done